libs/ng-utils/src/lib/use-http-image-source/use-http-image-source-pipe.module.ts

```diff
@@ -0,0 +1,28 @@
+import { ModuleWithProviders, NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { UseHttpImageSourcePipe } from './use-http-image-source.pipe';
+
+@NgModule({
+  imports: [CommonModule],
+  declarations: [UseHttpImageSourcePipe],
+  exports: [UseHttpImageSourcePipe],
+})
+export class UseHttpImageSourcePipeModule {
+  static forRoot(
+    config: { loadingImagePath?: string; errorImagePath?: string } = {}
+  ): ModuleWithProviders<UseHttpImageSourcePipeModule> {
+    return {
+      ngModule: UseHttpImageSourcePipeModule,
+      providers: [
+        {
+          provide: 'THIS_DOT_LOADING_IMAGE_PATH',
+          useValue: config.loadingImagePath || null,
+        },
+        {
+          provide: 'THIS_DOT_ERROR_IMAGE_PATH',
+          useValue: config.errorImagePath || null,
+        },
```

Comment:
I think it's better to use `InjectionToken` here instead of raw strings. This way we make sure we don't create regression by doing a typo

Reply:
done